cmd/selfupdate.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of K9s
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/creativeprojects/go-selfupdate"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+func selfUpdate() *cobra.Command {
+	command := cobra.Command{
+		Use:   "selfupdate",
```

Comment:
To me selfUpdate has the connotation of updating while the app is running. 
Perhaps `update` or `upgrade` might be more appropriate?
Also what should happen if a user has current k9s session(s) up and running while updating the binary?

Reply:
Soooo, this is where holywar starts :):
1. [Homebrew](https://docs.brew.sh/Manpage#update-options) uses `update` command to update brew, and `upgrade` to update packages. 
2. [Composer](https://getcomposer.org/doc/03-cli.md#self-update-selfupdate) uses `self-update`
3. [Helm](https://helm.sh/docs/helm/helm_upgrade/#helm-upgrade) doesn't have selfupdate command, but they use `upgrade` cli to update helm releases.
4. Pip uses `pip install --upgrade pip` to update pip itself. 
5. ....

So, this is really hard to pick one command, but at the same time - I don't want to have multiple commands to achieve only one simple result :)

I'm okay to change this command to `update`.